tools/isobuild/linker.js

```diff
@@ -1044,6 +1044,10 @@ export var fullLink = Profile("linker.fullLink", function (inputFiles, {
     packageVariables: _.union(assignedVariables, declaredExports)
   });
 
+  if (includeSourceMapInstructions) {
+    header = SOURCE_MAP_INSTRUCTIONS_COMMENT + "\n\n" + header;
+  }
```

Comment:
Can you move this block down closer to the other lines you hoisted out of the `_.map`? In other words, just after the `var footer = getFooter({...});` call.